doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -29,15 +59,18 @@ Refer to the following guides for more details.
 
 Kaa requires the following third party components to be installed and configured. For more information refer to [required third party](../#third-party-components) list.
 
+<ul>
+<li style="list-style-type: none;">
 <ul class="nav nav-tabs">
-  <li class="active"><a data-toggle="tab" href="#Ubuntu">Ubuntu 14.04</a></li>
+  <li class="active"><a data-toggle="tab" href="#Ubuntu">Ubuntu 14.04/16.04</a></li>
   <li><a data-toggle="tab" href="#CentOS">CentOS 6.7</a></li>
 </ul>
 
 <div class="tab-content">
 <div id="Ubuntu" class="tab-pane fade in active" markdown="1" >
 
-### Ubuntu 14.04
+#### Ubuntu 14.04/16.04
```

Comment:
It's better to remove this heading since it is already present in the tab title.